build-openjdk8.sh

```diff
@@ -1,13 +1,51 @@
 #!/bin/bash
 set -e
 
+# Determine platform name. Currently supported:
+#
+# x86_64 => x64_linux
+# aarch64 => aarch64_linux
+#
+platform_name() {
+  arch=$(uname -m)
+  case $arch in
+  x86_64)
+    echo "x64_linux"
+    ;;
+  aarch64)
+    echo "aarch64_linux"
+    ;;
+  *)
+    echo "Unsupported platform '$arch'" 1>&2
+    exit 1
+    ;;
+  esac
+}
+
+# Set the --with-boot-jdk based on platform
+boot_jdk() {
+arch=$(uname -m)
+  case $arch in
+  x86_64)
+    echo "/usr/lib/jvm/java-1.7.0-openjdk.x86_64"
+    ;;
+  aarch64)
+    echo "/usr/lib/jvm/java-1.7.0-openjdk-1.7.0.221-2.6.18.0.el7_6.aarch64"
+    ;;
+  *)
+    echo "Boot-JDK not fount platform '$arch'" 1>&2
+    exit 1
+    ;;
+  esac
+}
```

Comment:
This seems quite ugly and won't work if the version installed on the system isn't exactly the one you are referencing. Instead, lets use a more generic path which will exist irrespective of the underlying architecture. See below.